.env

```diff
@@ -0,0 +1 @@
+# Empty Environment File
```

Comment:
The `.env` file was removed from `.gitignore` since it's no longer relevant. You can go ahead and remove this file.

See https://github.com/thunder-app/thunder/pull/946 for more information!